index.js

```diff
@@ -209,6 +210,7 @@ class K8sExecutor extends Executor {
         this.buildTimeout = hoek.reach(options, 'kubernetes.buildTimeout') || DEFAULT_BUILD_TIMEOUT;
         this.maxBuildTimeout = this.kubernetes.maxBuildTimeout || MAX_BUILD_TIMEOUT;
         this.serviceAccount = this.kubernetes.serviceAccount || 'default';
+        this.dnsPolicy = this.kubernetes.dnsPolicy || 'ClusterFirst';
```

Comment:
should this be ClusterFirst or Default?

Reply:
`ClusterFirst` is the default value for k8s, so kept it.

Reply:
👍